### ISSUE TITLE
As per the guidlines FAB should be below navigation drawer

### ISSUE THREAD

**Issue by SilverFoxA** [slidenerd/materialtest#7]
https://cloud.githubusercontent.com/assets/5139030/6666009/1516139a-cc06-11e4-9134-4d68d4af56ce.gif 

I don't think this is correct

**Comment by slidenerd** [comment]
Yes you are right, I took the wrong library, should have gone with the other one, now If I make another video , my codebase ll change and all the people watching that video are gonna cry foul

**Comment by SilverFoxA** [comment]
ok, but can't you just change the position i.e. the `frame layout` of the same
